src/components/Admin.tsx

```diff
@@ -36,16 +36,71 @@ const Admin = () => {
                 Disclaimer Statements (Global)
               </Button>
             </li>
-            <li>Department of Defense Activity Address Directory (DoDAAD)</li>
+            <li>
+              <Button
+                appearance="transparent"
+                onClick={() =>
+                  seturl(
+                    _spPageContextInfo.webAbsoluteUrl +
+                      "/Lists/DODAAC/AllItems.aspx"
+                  )
+                }
+              >
+                DODAACs
+              </Button>
+            </li>
           </ul>
         </>
       )}
       {program && programPerms.data && (
         <>
           <Title3>{program} Program Office Items</Title3>
           <ul>
-            <li>Contractors</li>
-            <li>Contracts</li>
+            <li>
+              <Button
+                appearance="transparent"
+                onClick={() =>
+                  seturl(
+                    _spPageContextInfo.webAbsoluteUrl +
+                      "/" +
+                      program +
```

Comment:
Should we make this a const, such as programAbsoluteUrl, and then use that for each of the different links for Program Data?